lib/auth/signup/sign_up_user.dart

```diff
@@ -0,0 +1,15 @@
+
+import 'package:chat_app_front/auth/user.dart';
+import 'package:chat_app_front/auth/user_repository.dart';
+import 'package:dartz/dartz.dart';
+
+import '../../common/failure.dart';
+
+class SignUpUser{
```

Comment:
Co to za popelyna? Klasa nazywa się jak metoda, nie do końca wiadomo po co właściwie jest